lib/main.coffee

```diff
@@ -2,16 +2,16 @@ helpers = null
 
 module.exports =
   config:
-    pep8ExecutablePath:
+    pycodestyleExecutablePath:
```

Comment:
Let's just standardize this on `executablePath` like all the other linter providers are using. Prepending the name just adds more to type when working on the code and is meaningless as the setting is already tagged with the package name 😛.